app/details/[ticket_id]/ticketDetails.tsx

```diff
@@ -707,31 +707,36 @@ function TicketDetails(props: Props) {
                         })(),
                       }}
                     />
-                    <div className='tag-div'>
-                      <Icon
-                        iconName='sticky-note-icon'
-                        iconSize='12'
-                        iconViewBox='0 0 12 12'
-                        size={true}
-                        onClick={handleMacroItem}
-                        isActive={true}
-                      />
-                      {macroDropdown && (
-                        <DropDown
-                          items={macros}
-                          labelField='title'
-                          onClose={handleOutsideClick}
-                          onChange={handleMacroSelect}
-                          iconSize={''}
-                          iconViewBox={''}
-                          style={{ bottom: 60, maxWidth: 146, width: '100%' }}
-                          isMacro={
-                            currentWorkspace?.role === UserRole.OWNER ||
-                            currentWorkspace?.role === UserRole.ADMIN
-                          }
-                        />
-                      )}
-                    </div>
+                    {macros.length > 0 && (
```

Comment:
Here condition will be like
{(currentWorkspace?.role === UserRole.OWNER ||
                          currentWorkspace?.role === UserRole.ADMIN || macros.length > 0 && (

Reply:
Done